.eslintrc.js

```diff
@@ -0,0 +1,234 @@
+module.exports = {
+	ecmaFeatures : {
+		jsx     : true,
+		modules : true,
+	},
+	env : {
+		browser : true,
+		node    : true,
+		mocha   : true,
+		"es6": true,
```

Comment:
In ESLint 2.x, `parserOptions` was introduced where you can specify `ecmaVersion` ([docs](http://eslint.org/docs/user-guide/configuring)).